ledger-api/grpc-definitions/com/daml/ledger/api/v1/in_flight_transfers_service.proto

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1;
+
+import "com/daml/ledger/api/v1/event.proto";
+import "com/daml/ledger/api/v1/transaction_filter.proto";
+import "com/daml/ledger/api/v1/transfer.proto";
+
+option java_outer_classname = "InFlightTransfersServiceOuterClass";
+option java_package = "com.daml.ledger.api.v1";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1";
+
+// Allows clients to get in-flight state of transfers from the ledger.
```

Comment:
Update terminology to "incomplete" rather than "in-flight" following our recent discussions ?

Reply:
This will be unified with the work here: https://github.com/DACH-NY/canton/pull/12863#pullrequestreview-1416891300